src/index.js

```diff
@@ -43,9 +43,18 @@ AIOCommand.run = async (argv, opts) => {
       break
     }
   }
-
   // the second parameter is the root path to the CLI containing the command
-  return run(argv, config.options)
+  try {
+    return await run(argv, config.options)
+  } catch (error) {
+    // oclif throws if the user typed --help ... ?
+    if (error.oclif && error.oclif.exit === 0) {
+      await config.runHook('postrun')
```

Comment:
Why do we call a 'postrun' in case of oclif error? I assume the hook should run only if command executed successfully else  'command_error' hook should run.

Reply:
This is not really an error, oclif throws if the user typed `--help`